src/auth/awss3/core/qgsauthawss3method.cpp

```diff
@@ -39,8 +39,11 @@ QMap<QString, QgsAuthMethodConfig> QgsAuthAwsS3Method::sAuthConfigCache = QMap<Q
 QgsAuthAwsS3Method::QgsAuthAwsS3Method()
 {
   setVersion( 4 );
-  setExpansions( QgsAuthMethod::NetworkRequest );
-  setDataProviders( QStringList() << QStringLiteral( "awss3" ) );
+  setExpansions( QgsAuthMethod::NetworkRequest | QgsAuthMethod::DataSourceUri );
+  setDataProviders( QStringList()
+  << QStringLiteral( "awss3" )
+  << QStringLiteral( "ogr" )
```

Comment:
Is it really possible to access a file from an S3 platform with ogr provider? gdal is not enough here?